compiler/src/iree/compiler/Codegen/SPIRV/MaliConfig.cpp

```diff
@@ -27,9 +26,15 @@ static LogicalResult setMaliMatmulConfig(linalg::LinalgOp op,
   const int subgroupSize = limits.getSubgroupSize();
   const std::array<int64_t, 2> workgroupXY = {subgroupSize / 2, 2};
   std::array<int64_t, 3> threadMNK;
-  auto inputType = op.getDpsInputOperand(0)->get().getType().cast<ShapedType>();
-  if (inputType.getElementType().getIntOrFloatBitWidth() == 16) {
+  Type elementType = op.getDpsInputOperand(0)
```

Comment:
Nit: can we use two local variables for this long chain of method calls? The formatting here is not very nice. :)